.gitattributes

```diff
@@ -0,0 +1,7 @@
+###############################################################################
+# Set default behavior to automatically normalize line endings.
```

Comment:
```suggestion
# Never normalize line endings.
```

Reply:
behavior is the US English spelling of the word. Is there a policy to use British English in the repo?